features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/mysql.sql

```diff
@@ -1287,6 +1287,32 @@ CREATE TABLE IDN_OAUTH_PAR (
     PARAMETERS MEDIUMTEXT
 )ENGINE INNODB;
 
+CREATE TABLE IDN_ORG_USER_INVITATION (
+	ID INTEGER NOT NULL AUTO_INCREMENT,
+	INVITATION_ID VARCHAR(40) NOT NULL,
+	CONFIRMATION_CODE VARCHAR(40) NOT NULL,
+	USER_NAME VARCHAR(255) NOT NULL,
+	DOMAIN_NAME VARCHAR(255) NOT NULL,
+	EMAIL VARCHAR(255) NOT NULL,
+	USER_ORG_ID VARCHAR(255) NOT NULL,
+	INVITED_ORG_ID VARCHAR(255) NOT NULL,
+	USER_REDIRECT_URL VARCHAR(1024) NOT NULL,
+	STATUS VARCHAR(10) NOT NULL,
+	CREATED_AT TIMESTAMP NOT NULL,
+	EXPIRED_AT TIMESTAMP NOT NULL,
+	PRIMARY KEY (INVITATION_ID)
+)ENGINE INNODB;
```

Comment:
`Incorrect table definition; there can be only one auto column and it must be defined as a key`

Reply:
This is a new table addition and we are in the process of testing this to mysql.